src/views/Header.vue

```diff
@@ -0,0 +1,105 @@
+
+
+<script>
+import {getAuthenticatedHeaders,logout} from "@/static_functions";
+export default {
+  emits: ['tendersOpen', 'offersOpen', 'newTenderOpen'],
+  data() {
+    return {
+      page: 0,
+      tenderCount: 0,
+      offerCount: 0,
+    }
+  },
+  async beforeCreate() {
```

Comment:
api calls should be placed in beforeMount or mounted hooks. using fetch inside beforeCreate can slow down the component